### ISSUE TITLE
Read Holding Registers

### ISSUE THREAD

**Issue by alienatedsec** [jmccrohan/ha_pyscript_pysolarmanv5#1]
Thanks for the integration @jmccrohan 

I am trying to read some registers but wondering where the output should be.
![image](https://user-images.githubusercontent.com/73167064/198847115-18086a29-6bd8-4c40-8724-a0977eb40956.png)

Apparently, it runs successfully.

**Comment by jmccrohan** [comment]
For the moment, the read services do nothing only write to the HA logfile.
https://github.com/jmccrohan/ha_pyscript_pysolarmanv5/blob/main/ha_pyscript_pysolarmanv5/__init__.py#L59

A HA service cannot return any result by itself, so I need to figure out where the best place to store the result is. Perhaps passing a sensor value as a third parameter and writing the result there.

**Comment by alienatedsec** [comment]
So I had no results in the logs for read services, but error for trying to write

```
This error originated from a custom integration.

Logger: custom_components.pyscript.apps.ha_pyscript_pysolarmanv5.pysolarmanv5_write_holding_register
Source: custom_components/pyscript/eval.py:478
Integration: Pyscript Python scripting (documentation, issues)
First occurred: 07:24:49 (1 occurrences)
Last logged: 07:24:49

Failed to write_holding_register(43110, 35
```
The last line looks as copied from the log window

![image](https://user-images.githubusercontent.com/73167064/198867353-4411a111-197b-496e-aa71-fb0f0bde9a3c.png)

**Comment by jmccrohan** [comment]
@alienatedsec 

> So I had no results in the logs for read services, but error for trying to write

This should be resolved now with bad7894. I omitted to copy over the latest version of the file from my dev HA instance before committing.
